src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -128,6 +128,46 @@ public async Task<IActionResult> GetRecommendations(string sessionId)
             return Ok(output);
         }
 
+        /// <summary>
+        /// Gets a summary of the Recipe for the session's project.
+        /// </summary>
+        [HttpGet("session/<sessionId>/recipe/<recipeId>")]
+        [SwaggerOperation(OperationId = "GetRecipe")]
+        [SwaggerResponse(200, type: typeof(RecipeSummary))]
+        [Authorize]
+        public IActionResult GetRecipe(string sessionId, string recipeId)
+        {
+            var state = _stateServer.Get(sessionId);
+            if (state == null)
+            {
+                return NotFound($"Session ID {sessionId} not found.");
+            }
+
+            if (string.IsNullOrEmpty(recipeId))
+            {
+                return NotFound($"A Recipe ID was not provided.");
+            }
+
+            var recommendation = state.NewRecommendations?.FirstOrDefault(x => x.Recipe.Id.Equals(recipeId));
+
+            if (recommendation == null)
+            {
+                return NotFound($"Recipe ID {recipeId} not found.");
+            }
+
+            var output = new RecipeSummary(
+                recommendation.Recipe.Id,
+                recommendation.Recipe.Version,
+                recommendation.Recipe.Name,
+                recommendation.Recipe.Description,
+                recommendation.Recipe.TargetService,
+                recommendation.Recipe.DeploymentType.ToString(),
+                recommendation.Recipe.DeploymentBundle.ToString()
```

Comment:
Any reason behind converting the enum to string? enumerations are supported in JSON.

Reply:
Leaving them as enums blows up from the toolkit's side when it's trying to deserialize them.